lib/elixir/lib/io/ansi/docs.ex

```diff
@@ -291,6 +291,8 @@ defmodule IO.ANSI.Docs do
     col =
       col
       |> String.replace("\\\|", "|")
+      |> String.replace(~r/-+/, "-")
```

Comment:
Hrm, it will. I think we need to do this squashing only after we detect the first line is a table_header?